flatgfa/src/ops/gaf.rs

```diff
@@ -191,6 +192,46 @@ impl ChunkEvent {
             ChunkRange::None => {}
         }
     }
+    pub fn get_seq_string(&self, gfa: &flatgfa::FlatGFA) -> String {
+        let seq = gfa.get_seq_oriented(self.handle);
+    
+        match self.range {
+            ChunkRange::Partial(start, end) => seq.slice(start..end).to_string(),
+            ChunkRange::All => seq.to_string(),
+            ChunkRange::None => String::new(),
+        }
+    }
+    pub fn get_seg(&self, gfa: &flatgfa::FlatGFA)->String {
```

Comment:
Similarly, this duplicates code with the existing `print` method. Please combine them into one implementation, with one method calling the other… or maybe we just want the version that returns a string, and the client can just use `println!` on that if it wants to print.